src/ui/compilationStatusView.ts

```diff
@@ -36,6 +40,7 @@ function toStatusMessage(status: CompilationStatus, message?: string | null): st
       ? `${Messages.CompilationStatus.VerificationFailedOld} ${message}`
       : `${Messages.CompilationStatus.VerificationFailedOld}`;
   }
+  return status.toString();
```

Comment:
This is for future forward compatibility, rivht?

Reply:
I don't think it's needed for the current changes, but I thought it might be good defensive programming